lib/std/vmutils.nim

```diff
@@ -9,3 +9,14 @@ proc vmTrace*(on: bool) {.compileTime.} =
       var a = 1
       vmTrace(false)
     static: fn()
+
+proc debugNimNode*(a: NimNode): string =
+  ## Implementation-specific rendering of `a` in the compiler, unstable.
+  runnableExamples:
```

Comment:
Comment or document where you can find the implementation of debugNimNode?

Reply:
done